apps/web/app/api/ably-auth/route.ts

```diff
@@ -0,0 +1,34 @@
+import { isProd } from "@/constants/contracts";
+import { CHANGE_EVENT_CHANNEL_NAME } from "@/globals";
+import Ably from "ably";
+import { NextResponse, NextRequest } from "next/server";
+
+export async function POST(req: NextRequest, res: NextResponse) {
+  // Used for linter that fails
+  if (!process.env.NEXT_ABLY_API_KEY) {
+    console.error("NEXT_ABLY_API_KEY env must be");
+    return NextResponse.json({
+      status: 200,
+      message: "No auth",
```

Comment:
should be internal error instead?

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/2bf171b7ee6ab8ceb0b5db67d50e02f329edb829

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/2bf171b7ee6ab8ceb0b5db67d50e02f329edb829

Reply:
did new comment there

Reply:
Where ? In the commit link I dont see

Reply:
Found it